lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -364,24 +371,26 @@ def cmd_search_help
             print_line
             print_line "Examples:"
             print_line "  search cve:2009 type:exploit"
+            print_line "  search cve:2009 type:exploit platform:-linux"
             print_line
           end
 
           #
           # Searches modules for specific keywords
           #
           def cmd_search(*args)
-            match       = ''
-            search_term = nil
-            output_file = nil
-            cached      = false
-            use         = false
-            count       = -1
+            match        = ''
+            row_filter  = nil
```

Comment:
What are your thoughts on calling this `search_term` as it previously was, and renaming `search_terms` to highlight strings?

Reply:
I think the current naming scheme is clearer. If we changed it back then the regex pattern we use to filter rows would be called `search_term` and the words used to search MS would be called `highlight_strings` internally, which isn't as readable  IMHO

Reply:
Yeah, the naming's interesting. After learning _what_ this does, I agree that this name is clearer. It's a shame the rex table api uses `SearchTerm`